helm/frconfig/README.md

```diff
@@ -1,41 +1,55 @@
-is suitable for this purpose.
+This is a prerequesite chart that must be deployed before other charts such as openig, amster, and openidm.
 
 ## values.yaml
 
-The defaults provided in values.yaml will clone the "public" forgeops-init git repository. This is a bare bones
-starter repository with a minimal platform configuration. 
+The defaults in values.yaml clone the public (read only) [forgeops-init](https://github.com/ForgeRock/forgeops-init) repository. This 
+is a bare bones starter repository with a minimal platform configuration. A https: git URL is used to clone the configuration, and
+it does not require any git ssh secrets to perform the pull.  
+
+To use a different git repository, you must update values.yaml with the appropriate git URLs. 
+Use a git:// url for private repostories that require ssh access, as http only works for public, read-only repositories.
 
-If you want to use a custom configuration, create a custom.yaml file that overrides the defaults. A sample
+To use a custom configuration, create a custom.yaml file that overrides the defaults. A sample
 is shown below:
 
 ```yaml
 git:
   # git repo to clone.
   repo: "git@github.com:Acme/cloud-deployment-config.git"
-  branch: master
-  # Name of a secret that contains a key "id_rsa"
-  # The secret contains the git ssh key that has permissions to clone and/or update the git repo.
-  sshSecretName: "git-ssh-key"
+  branch: mybranch
+# Usually you do not need to change config.name. See below
+# config:
+#   name: frconfig
+```
+
+## git secret
+
+A dummy ssh secret `id_rsa` is stored in the `frconfig` secret. If you need ssh access to your git repository
+you must replace this secret with a real ssh key. For example:
+
+```shell
+# Generate your own id_rsa and id_rsa.pub keypair, according to the instructions on github or stash.
+kubectl delete secret frconfig
+kubectl create secret generic frconfig --from-file=id_rsa
 ```
 
-If you use a custom git repository make sure you create an ssh secret that contains a key `id_rsa`. This is the private key that has permissions to clone and/or update your repository (the public part of this key is uploaded to your github or stash repository).  Set the sshSecretName to the name of this secret. 
+Note the secret name *must* be id_rsa.  This is the private key that has permissions to clone and/or update your repository (the public part of this key is uploaded to your github or stash repository).
 
 ## Configuration per product
 
-This project uses a single git repository that contains configuration for all products. If you want to use a strategy of a configuration repository per product, you can deploy multiple instances of this chart. For each instance you must customize:
+This project uses a single git repository that contains configuration for all products. If you want to use a strategy of a configuration repository per product, you can deploy multiple instances of this chart, each with a different name for `config.name`.
```

Comment:
Or, instead of "see the comments below" before the config.name key, you could move these comments so that they are close to the config.name key. (For that matter, why not move config.name to the end of the values.yaml file, right under this big comment?